apps/browser/src/connectors/sso.ts

```diff
@@ -0,0 +1,26 @@
+﻿document.addEventListener("DOMContentLoaded", () => {
+  const lastPass = getQsParam("lp");
```

Comment:
```suggestion
if (document.readyState === "loading") {
  document.addEventListener("DOMContentLoaded", loadSsoConnector);
} else {
  loadSsoConnector();
}

function loadSsoConnector() {
  const lastPass = getQsParam("lp");
```

Similar to the implementation within `autofiller.ts`, a race condition can be avoided here where if the script somehow parses after the DOMContentLoaded event has triggered, it will still initialize as expected.